src/entities/movie/model/types.ts

```diff
@@ -6,3 +6,36 @@ export interface TrendedMovie {
     duration: number;
     genre: string;
 }
+
+export interface Movie {
+    id: string;
+    createdAt: Date;
+    updatedAt: Date;
+    title: string;
+    description: string;
+    duration: number;
+    imageUrl: string;
+    genre: string;
+    releaseDate: Date;
+    trended: boolean;
+}
+
+export interface PaginatedMovies {
```

Comment:
I'm pretty sure that we'll use the same interface structure for all paginated models in future 
So let's create a shared interface PaginatedModel and use it here 
```
{
  pages: { 
    items: T[]
    meta: {...}
    links: {...}
  }
  pageParams: number[];
}
```